.github/workflows/build-images.yaml

```diff
@@ -48,7 +40,7 @@ jobs:
           tags: ${{ env.IMG_TAGS }}
           platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
           build-args: |
-            version=${{ env.VERSION }}
+            GITHUB_SHA=${{ github.sha }}
```

Comment:
Why "GITHUB_SHA"? It should be "GIT_SHA", no?

Reply:
I can see why this would make sense, Gui. If it was a local deployment, `git` is how version control is managed, and even if on GitHub, the version control is also done with git. 

The reason why I wrote the variable as `GITHUB_SHA` was simply a foobar mistake. :) I will also make changes to have the name reflect its context. :)